src/XMakeBuildEngine/Construction/Solution/SolutionProjectGenerator.cs

```diff
@@ -104,6 +112,11 @@ internal class SolutionProjectGenerator
         private ILoggingService _loggingService;
 
         /// <summary>
+        /// The list of targets specified to use.
+        /// </summary>
+        private readonly IReadOnlyCollection<string> _targetNames = new Collection<string>();
```

Comment:
Ah, not all paths write to this fields. Ignore.

---
In reply to: [93343487](https://github.com/Microsoft/msbuild/pull/1497#discussion_r93343487) [](ancestors = 93343487)